src/subcommand/wallet/create.rs

```diff
@@ -1,8 +1,56 @@
-use super::*;
+use {
+  super::*,
+  bitcoin::secp256k1::rand::RngCore,
+  bitcoin::util::bip32::{ChildNumber, DerivationPath, ExtendedPrivKey},
+  bitcoincore_rpc::bitcoincore_rpc_json::{ImportDescriptors, Timestamp},
+  miniscript::descriptor::{Descriptor, DescriptorSecretKey, DescriptorXKey, Wildcard},
+};
 
 pub(crate) fn run(options: Options) -> Result {
+  // TODO: check for correct bitcoin core version
+
+  options
+    .bitcoin_rpc_client_mainnet_forbidden("ord wallet create")?
+    .create_wallet("ord", None, Some(true), None, None)?;
+
+  let secp256k1 = bitcoin::secp256k1::Secp256k1::new();
+
+  let mut seed = [0; 32];
+  bitcoin::secp256k1::rand::thread_rng().fill_bytes(&mut seed);
+
+  let xkey = ExtendedPrivKey::new_master(options.chain().network(), &seed)?;
+
+  let secret_key = DescriptorSecretKey::XPrv(DescriptorXKey {
+    origin: Some((
+      xkey.fingerprint(&secp256k1),
+      DerivationPath::master()
+        .child(ChildNumber::Hardened { index: 86 })
+        .child(ChildNumber::Hardened { index: 0 })
+        .child(ChildNumber::Hardened { index: 0 }),
+    )),
+    xkey,
+    derivation_path: DerivationPath::master().child(ChildNumber::Normal { index: 0 }),
+    wildcard: Wildcard::Unhardened,
+  });
```

Comment:
The descriptor now looks correct, but it doesn't actually reflect how it was derived.

The origin shows a derivation path, but the xkey is still the original, master, underived xkey. We need to use a derived key, like so:

```suggestion
  let master_xkey = ExtendedPrivKey::new_master(options.chain().network(), &seed)?;

  let fingerprint = xkey.fingerprint(&secp256k1)
  
  let derivation_path = DerivationPath::master()
        .child(ChildNumber::Hardened { index: 86 })
        .child(ChildNumber::Hardened { index: 0 })
        .child(ChildNumber::Hardened { index: 0 });
        
  let derived_xkey = master_xkey.derive_priv(secp256k1, derivation_path)?;

  let secret_key = DescriptorSecretKey::XPrv(DescriptorXKey {
    origin: Some((fingerprint, derivation_path)),
    xkey: derived_xkey,
    derivation_path: DerivationPath::master().child(ChildNumber::Normal { index: 0 }),
    wildcard: Wildcard::Unhardened,
  });
```